file = open('during/error.txt')

file_content = file.read()

print(file_content)